irr/include/Keycodes.h

```diff
@@ -5,9 +5,6 @@
 #pragma once
 #include <variant>
 
-namespace irr
-{
-
 enum EKEY_CODE
```

Comment:
I figure we might want to turn this into an enum class or wrap it in something like a new `key` namespace at least; though the `KEY_` enum name prefix should also suffice.

Reply:
`EKEY_CODE` can be dropped (or replaced by a `const unordered_map<string, SDL_Keycode>` for compatibility with older configurations) once we fully move to SDL, so I don't think we need to worry too much about this.